roles/redfish_monitor/templates/redfish-monitor.env.j2

```diff
@@ -0,0 +1,12 @@
+OS_AUTH_STRATEGY=keystone
+OS_AUTH_URL={{ keystone_internal_url }}/v3
```

Comment:
Make sure that internal_url isn't suffixed with /v3, this has been a problem in the past I think.